scripts/deploy-fw.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+SUPPORTED_PLATFORMS=(byt cht bdw hsw apl cnl sue icl skl kbl)
```

Comment:
No need for this, as there is no harm in copying all FW binaries to every host (this will be typical distro install)

Reply:
I think this is a better way to make sure only the needed binary is copied. To make sure we are working with the right file.